src/main/java/com/conveyal/gtfs/loader/JdbcTableWriter.java

```diff
@@ -1568,10 +1569,41 @@ private void updateReferencingTables(
                             // FIXME: is this where a delete hook should go? (E.g., CalendarController subclass would override
                             //  deleteEntityHook).
                             if (sqlMethod.equals(SqlMethod.DELETE)) {
+                                ArrayList<String> patternAndRouteIds = new ArrayList<String>();
```

Comment:
Use just the diamond operator (`new ArrayList<>()`).

Reply:
Updated in https://github.com/conveyal/gtfs-lib/pull/378/commits/30754ba741ee414fc1778837160d91e12694eddf